.github/workflows/sdk-cli-perf-monitor-test.yml

```diff
@@ -33,6 +33,12 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
+      - name: set test mode
```

Comment:
I had a PR earlier to remove these 2 steps, replaced by setting it in `env` (this does not work well in Windows), so please remove these two steps to keep it same with current main branch.

Reply:
reference: https://github.com/microsoft/promptflow/pull/1532